### DISCUSSION TITLE
"@user is typing..." would be cool.

### DISCUSSION THREAD

**Discussion by mktcode** [community/community#10309] [Projects and Issues]
Everyone knows that from their favorite chat app. Would be helpful for GitHub too. I personally had quite a few situations where this would have saved me or others from writing pointless comments.

Something like this:

![image](https://user-images.githubusercontent.com/6792578/150197021-9a876be9-8548-4d49-8caa-cad526a43bbd.png)

**Comment by metrox-eth** [comment]
I second that

**Comment by samwhosung** [comment]
How do we add this?

**Reply by mktcode** [reply]
Let's leave that to github... :P

**Comment by ChaituVR** [comment]
This would be a great feature
